windows/src/desktop/setup/UfrmRunDesktop.pas

```diff
@@ -587,7 +587,8 @@ procedure TfrmRunDesktop.FillLanguageList;
   item: TPair<string,string>;
   n: Integer;
 begin
-  for item in TSetupUILanguageManager.Locales do
+  TSetupUILanguageManager.Reorder;
```

Comment:
This should be in the `Locales` getter function, keeping ordering better encapsulated.